examples/drivenet/drivenet_open_loop.ipynb

```diff
@@ -0,0 +1,354 @@
+    "    \n",
+    "    predicted_positions = transform_points(predicted_positions, data[\"raster_from_agent\"])\n",
+    "    target_positions = transform_points(target_positions, data[\"raster_from_agent\"])\n",
+    "    \n",
+    "    draw_trajectory(im_ego, predicted_positions, PREDICTED_POINTS_COLOR)\n",
+    "    \n",
+    "    draw_trajectory(im_ego, target_positions, TARGET_POINTS_COLOR)\n",
+    "\n",
+    "    plt.imshow(im_ego[::-1])\n",
+    "    plt.axis(\"off\")\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Qualitative Evaluation: Visualise the Open Loop\n",
+    "\n",
+    "To visualise the open loop we can just repeat the same operations for consecutives frames.\n",
+    "\n",
+    "In this example, we show the first 200 frames for our dataset, plotting predicted and annotated trajectories.\n",
+    "\n",
+    "**Note: this is an open loop evaluation, we are NOT controlling the AV with our predictions**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import display, clear_output\n",
+    "import PIL\n",
+    " \n",
+    "for frame_number in range(200):\n",
+    "    \n",
+    "    data = eval_dataloader.dataset[frame_number]\n",
+    "    del data[\"host_id\"]\n",
+    "\n",
+    "    data_batch = default_collate([data])\n",
+    "    \n",
+    "    result = model(data_batch)\n",
+    "    predicted_positions = result[\"positions\"].detach().cpu().numpy().squeeze()\n",
+    "\n",
+    "    \n",
+    "    predicted_positions = transform_points(predicted_positions, data[\"raster_from_agent\"])\n",
+    "    target_positions = transform_points(data[\"target_positions\"], data[\"raster_from_agent\"])\n",
+    "    \n",
+    "    im_ego = rasterizer.to_rgb(data[\"image\"].transpose(1, 2, 0))\n",
+    "    draw_trajectory(im_ego, target_positions, TARGET_POINTS_COLOR)\n",
+    "    draw_trajectory(im_ego, predicted_positions, PREDICTED_POINTS_COLOR)\n",
+    "    \n",
+    "    clear_output(wait=True)\n",
+    "    display(PIL.Image.fromarray(im_ego[::-1]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Is open loop evaluation enough?\n",
+    "\n",
+    "Depending on how long you've trained your model, you may have seen different results in the above cell.\n",
+    "\n",
+    "If you've trained it long enough, the predicted trajectory is likely well overlapped with the annotated one.\n",
+    "\n",
+    "Conversely, a model trained not enough will surely show some bias or unfeasible trajectories.\n",
+    "\n",
+    "In both cases, **this evaluation is not enough** to ensure your model will be able to actually drive on the road (that's where we all want to go in the end). If your model is not in controll of where to go, you can't really say it will work once the annotated trajectory won't be available anymore.\n",
```

Comment:
```suggestion
    "In both cases, **this evaluation is not enough** to ensure your model will be able to actually drive on the road (that's where we all want to go in the end). If your model is not in control of where to go, you can't really say it will work once the annotated trajectory won't be available anymore.\n",
```